cmd/bios/set.go

```diff
@@ -0,0 +1,30 @@
+package bios
+
+import (
+	"log"
+
+	mctl "github.com/metal-toolbox/mctl/cmd"
+	rctypes "github.com/metal-toolbox/rivets/condition"
+	"github.com/spf13/cobra"
+)
+
+var setCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set BIOS settings to something else",
```

Comment:
I think we could come up with a nicer description than, "change to something else". Something like "Set BIOS settings from remote config file".